api/v1/openapi.yaml

```diff
@@ -146,6 +146,8 @@ paths:
       responses:
         '201':
           description: Created
+          schema:
+            "$ref": "#/definitions/EndpointID"
```

Comment:
Did you consider whether to reuse the existing `Endpoint` object?

It looks like there are no required fields for that object type, so you should be able to construct a response similar to your current solution with only the endpoint ID. I think that this would intuitively make sense to have the response to the endpoint creation be an endpoint object.

The benefit would be that you could drop the `EndpointID` object, which would keep the API simpler.